docs_src/microservices/application/BuiltIn.md

```diff
@@ -126,9 +126,11 @@ There is one Core Data function that enables interactions with the Core Data RES
     NewCoreDataSimpleReading("my-profile", "my-device", "my-resource", "string").PushToCoreData
     ```
 
-## Encryption
+## Data Protection
 
-There is one encryption transform included in the SDK that can be added to your pipeline. 
+There are two transforms included in the SDK that can be added to your pipeline for data protection. 
+
+### Encryption (Deprecated)
 
 | Factory Method                                               | Description                                                  |
```

Comment:
Add caution note that this is deprecated in 2.1 and to use AESProtection

Reply:
Can we send strong messaging than that?  "Please read security advisory posted to app-functions-sdk-go before using this function."?

Reply:
@AlexCuse Unfortunately, I can't publish the advisory until we get the Jakarta release out.  So no link yet.

Reply:
got it - can leave a placeholder in docs PR for now 👍

Reply:
```
!!! edgey "EdgeX 2.1"
    This is deprecated in EdgeX 2.1 - it is recommended to use the new `AESProtection` transform.  Please see [this security advisory](#TODO) for more detail.

```